### PR TITLE
[VEN-2713]: Deploy Liquid Staked ETH pool to arbitrum one 

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -4499,6 +4513,84 @@ export const globalConfig: NetworkConfig = {
...
+            symbol: "vwstETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.09", 18),
+            jumpMultiplierPerYear: convertToUnit("3", 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.93", 18),
+            liquidationThreshold: convertToUnit("0.95", 18),
+            reserveFactor: convertToUnit("0.25", 18),
+            initialSupply: convertToUnit(2, 18),
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4535]:
```suggestion
            initialSupply: convertToUnit(3.55, 18),
```

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -4499,6 +4513,84 @@ export const globalConfig: NetworkConfig = {
...
+            symbol: "vweETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.09", 18),
+            jumpMultiplierPerYear: convertToUnit(3, 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.93", 18),
+            liquidationThreshold: convertToUnit("0.95", 18),
+            reserveFactor: convertToUnit("0.25", 18),
+            initialSupply: convertToUnit(2, 18),
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4553]:
```suggestion
            initialSupply: convertToUnit(4, 18),
```

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -4499,6 +4513,84 @@ export const globalConfig: NetworkConfig = {
...
+            symbol: "vWETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.035", 18),
+            jumpMultiplierPerYear: convertToUnit("0.8", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.77", 18),
+            liquidationThreshold: convertToUnit("0.8", 18),
+            reserveFactor: convertToUnit("0.2", 18),
+            initialSupply: convertToUnit(2, 18),
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4571]:
```suggestion
            initialSupply: convertToUnit(1.9678, 18),
```

File: helpers/deploymentConfig.ts [hunk 4]
```diff
@@ -4499,6 +4513,84 @@ export const globalConfig: NetworkConfig = {
...
+            jumpMultiplierPerYear: convertToUnit("3", 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.93", 18),
+            liquidationThreshold: convertToUnit("0.95", 18),
+            reserveFactor: convertToUnit("0.25", 18),
+            initialSupply: convertToUnit(2, 18),
+            supplyCap: convertToUnit(8_000, 18),
+            borrowCap: convertToUnit(800, 18),
+            reduceReservesBlockDelta: REDUCE_RESERVES_BLOCK_DELTA_ARBITRUM_ONE,
+            vTokenReceiver: preconfiguredAddresses.arbitrumone.VTreasury,
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4539]:
```suggestion
            vTokenReceiver: "0x5A9d695c518e95CD6Ea101f2f25fC2AE18486A61",
```

File: helpers/deploymentConfig.ts [hunk 5]
```diff
@@ -4499,6 +4513,84 @@ export const globalConfig: NetworkConfig = {
...
+            jumpMultiplierPerYear: convertToUnit(3, 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.93", 18),
+            liquidationThreshold: convertToUnit("0.95", 18),
+            reserveFactor: convertToUnit("0.25", 18),
+            initialSupply: convertToUnit(2, 18),
+            supplyCap: convertToUnit(4_600, 18),
+            borrowCap: convertToUnit(2_300, 18),
+            reduceReservesBlockDelta: REDUCE_RESERVES_BLOCK_DELTA_ARBITRUM_ONE,
+            vTokenReceiver: preconfiguredAddresses.arbitrumone.VTreasury,
```
Comment (chechu) [inline helpers/deploymentConfig.ts:4557]:
```suggestion
            vTokenReceiver: "0x46cba1e9b1e5db32da28428f2fb85587bcb785e7",
```

File: helpers/deploymentConfig.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts [hunk 6]
```diff
@@ -4499,6 +4513,84 @@ export const globalConfig: NetworkConfig = {
...
+            name: "Venus WETH (Liquid Staked ETH)",
+            asset: "WETH",
+            symbol: "vWETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.035", 18),
+            jumpMultiplierPerYear: convertToUnit("0.8", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.77", 18),
+            liquidationThreshold: convertToUnit("0.8", 18),
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 3 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 4 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

File: helpers/deploymentConfig.ts (hunk 5 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Fixed, 3dc571e86438747cd2695b1adca4f8f916ffdaad

**Pull request merged**